### PR TITLE
feat(masthead): v2 updates

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

#1627 
### Description

updates the website with new masthead changes.... stilll waiting for images

### Changelog

**New**

- created a folder for the masthead to get the tabs to correctly work

### CODE REVIEW

File: src/pages/components/masthead/level-0-l0.mdx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+**Mobile layout**
+
+On small and medium breakpoints, the mobile layout of the L0 masthead is displayed. A menu icon is shown at the left of the menu bar, along with the IBM logo and utility navigation items. On click of the menu icon, the open menu appears and displays one level of navigation items on screen, allowing the user to navigate back and forth through levels to find their desired destination.
+
+If a heading zone is used in a menu, both the label and description display on mobile. However, any supporting descriptions that are associated with link group titles or individual link items are suppressed at these breakpoints.
+
+<ResourceLinks name="Masthead" type="ui" />
+
+## Content specifications
```
Comment (oliviaflory) [inline src/pages/components/masthead/level-0-l0.mdx:144]:
```suggestion
## Content guidance
```

File: src/pages/components/masthead/level-1-l1.mdx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+Unlike the open L0 navigation that shows one level of content per screen view, the L1 masthead uses accordions to allow the user to access all navigation items contained in the menu. The user has full control over the accordion items and may choose to open and close each as suits their browsing behavior.
+
+Because the L1 section name acts as the toggle mechanism for opening and closing the menu, the open state introduces an “Overview” link item that allows access to the section’s main page.
+
+Note that if supporting descriptions are used with `link group titles` or `link list with descriptions`, this copy is suppressed at the mobile breakpoints.
+
+<ResourceLinks name="Masthead" type="ui" />
+
+## Content specifications
```
Comment (oliviaflory) [inline src/pages/components/masthead/level-1-l1.mdx:140]:
```suggestion
## Content guidance
```

**Review by RichKummer — COMMENTED** [review]
Grabbed a few typos going through the draft

**Review by oliviaflory — APPROVED** [review]
@ariellalgilmore this looks ready to go (if the link fix I just added works) !

**Pull request merged**